crates/uv/src/main.rs

```diff
@@ -698,6 +698,14 @@ struct PipListArgs {
     /// List editable projects.
     #[clap(short, long)]
     editable: bool,
+
+    // Exclude editable package from output.
+    #[clap(long)]
+    exclude_editable: bool,
```

Comment:
This is the default, right?

Reply:
By default all packages are listed (confirmed with `pip list` locally)